### DISCUSSION TITLE
Zim on Raspberry Pi 4 - version ?

### DISCUSSION THREAD

**Discussion by Nick222** [zim-desktop-wiki/zim-desktop-wiki#1606] [Q&A] [answered]
I install Zim on Raspberry Pi 4 - but it is only Zim 0.68 for it.

How can I install latest stable Zim version on Raspberry Pi 4 ?

**Comment by jaap-karssenberg** [comment, marked as answer]
Probably you need to include a bit of information on the distribution you
are running. If it is debian based, did you try the ".deb" package from the
website?  Else just go over the dependencies in the README file and try
running from source.

-- Jaap
